README.md

```diff
@@ -10,4 +10,47 @@ Each branch name corresponds to the last completed step in the code on the branc
 It's important to note that the code in this repo may differ from the code written on-the-spot in your training session. However, the files will contain the same information and represent a complete version of each concept.
 
 ## What next?
-To use this repo, clone the repo to your local machine and check out a new branch (any name is fine) from `master`. This is the starting point for [user story](https://docs.google.com/document/d/18W2lXAE9Y5HmidFF_b2zSv55lNSXREIJnLFhmGy_XRY/edit?usp=sharing) Fusion-01.
+If you are continuing this training from the last user story, then keep working on your current branch USERNAME-fusion-training.
+
+If you are starting this training from the second user story (task Fusion-02), then:
+
+1. Clone this repo
+```
+git clone git@github.com:wapopartners/Fusion-Training-User-Stories.git
+```
+
+2. Checkout to the fusion-01 branch (this will give you all the completed work for task Fusion-02)
```

Comment:
Not sure if we should specify that they will need to `cd` into the Fusion-Traning-User-Stories directory before they can `git checkout fusion-01` or if that is something that doesn't need to be spelled out?

Reply:
Lets spell it out or at least say, in the root of the repo...